packages/usa-in-page-navigation/src/test/in-page-navigation.spec.js

```diff
@@ -175,6 +180,40 @@ tests.forEach(({ name, selector: containerSelector }) => {
       assert(window.scroll.calledOnceWith(sinon.match({ top: 880 })));
     });
 
+    describe("Minimum heading count tests", () => {
+      beforeEach(() => {
+        body.innerHTML = TEMPLATE;
+        theNav = document.querySelector(THE_NAV);
+      });
+
+      afterEach(() => {
+        body.innerHTML = "";
+        window.location.hash = "";
+      });
+
+      it("does not render the in-page navigation when minimum heading count is too high", () => {
+        theNav.setAttribute("data-minimum-heading-count", "20");
+        behavior.off(containerSelector(body));
+        behavior.on(containerSelector(body));
+        assert.strictEqual(
+          theNav.hasChildNodes(),
+          false,
+          "In-page navigation should not have child nodes when the count is too high",
+        );
+      });
+
+      it("does render the in-page navigation when minimum heading count is low", () => {
```

Comment:
**question**: Confirming, this hides the in-page navigation when the minimum heading count is low right?

Reply:
I will reword this to be "does render the in-page navigation when default minimum heading count of 3 has not been reached"

Reply:
I see. The navigation still gets created, but is visually hidden because the minimum heading count is too low.

Reply:
Tests were updated in [fdbd228](https://github.com/uswds/uswds/pull/6205/commits/fdbd2281ac74231198d9b1521a0ce2f90427c250)